src/site/content/en/blog/index.njk

```diff
@@ -20,62 +20,10 @@ pagination:
   <div class="w-layout-container w-pb--non">
     <header class="w-page-header">
       <h1 class="w-page-header__headline">{{ title }}</h1>
-      <p class="w-page-header__copy">
-        Our latest news, updates, and stories for developers
-      </p>
+      <p class="w-page-header__copy">{{ description }}</p>
+      <div class="w-chips">
```

Comment:
Should `w-chips` be removed?

Reply:
`w-chips` has some styling which is relevant I think.

Reply:
I think in this instance, since we decided not to display the chips at the top of the page, this element is empty and could be removed.